kano_settings/home_screen.py

```diff
@@ -36,165 +20,102 @@ class UnknownScreenError(Exception):
 
 
 class HomeScreen(Gtk.Box):
+    _DISPLAYED_SCREENS = SCREENS.get_screens_on_home()
 
-    names = ["Keyboard", "Mouse", "Audio", "Display", "Wifi", "Overclocking",
-             "Account", "Appearance", "Font", "Advanced", "About",
-             "Notifications"]
-    custom_info = ["Keyboard-country-human", "Mouse", "Audio", None, None,
-                   "Overclocking", None, "Wallpaper", "Font"]
+    def __init__(self, win, screen_number=None, screen_name=None):
```

Comment:
No particular reason, I suppose that I thought that it might be easier to deprecate the `screen_number` if it were a separate parameter.

If you think that it would be better otherwise then I'm happy to change it.